app/schema/api/current/models.rb

```diff
@@ -155,7 +155,7 @@ def blood_sugar
         properties: {
           id: { '$ref' => '#/definitions/uuid' },
           blood_sugar_type: { type: :string, enum: BloodSugar.blood_sugar_types.keys },
-          blood_sugar_value: { type: :integer },
+          blood_sugar_value: { type: :decimal },
```

Comment:
There appears to be also be a `number` type in JSON schema. Is that any different from `decimal`?

Reply:
The `number` type was stale, re-swaggerized to update it